modules/serverless-install-web-console.adoc

```diff
@@ -5,11 +5,11 @@
 [id="serverless-install-web-console_{context}"]
 = Installing the {ServerlessOperatorName}
 
-This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the Operator Hub using the {product-title} web console.
```

Comment:
It's actually correct as "OperatorHub" one word (see the screenshot). Same for below change.